templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul class="sidebar-nav">
+  <li>
+    <a href="{% url "taxi:index" %}" style="color: #f5dd5d">Home page</a>
+  </li>
+  <li>
+    <a href="#" style="color: #f5dd5d">Manufacturers</a>
+  </li>
+  <li>
+    <a href="#" style="color: #f5dd5d">Cars</a>
+  </li>
+  <li>
+    <a href="#" style="color: #f5dd5d">Drivers</a>

```

Comment:
The links for 'Manufacturers', 'Cars', and 'Drivers' are currently placeholders (`href="#"`). If there are specific views or URLs for these sections, consider updating these links to use the `{% url %}` template tag to point to the correct paths.